### DISCUSSION TITLE
Doubts with Autoshoot Detection

### DISCUSSION THREAD

**Discussion by crashzk** [J-Tanzanite/Little-Anti-Cheat#109] [General]
I can't say if it would be a BUG or would it be like that, but if you configure CSGO to shoot by mouse scroll you get banned by aimbot, I tested it myself and really got banned by aimbot.

Very simple to test, just set the bind **`bind mwheeldownn +attack`** and play for a few minutes, done, banned by use of aimbot.

My question would be, this is classified as ban by aimbot, wouldn't it be better as macro and such?

**Comment by J-Tanzanite** [comment]
https://github.com/J-Tanzanite/Little-Anti-Cheat/issues/6 and https://github.com/J-Tanzanite/Little-Anti-Cheat/issues/6#issuecomment-624283002

So no, it's not a bug.
Autoshoot detection is just checking for perfect 1tick shots, which a LOT of cheats will do, a lot of them even with legit aimbot settings. While possible to do without cheating, it's rare; unless you try really hard or use specific binds like that... But if people have to go out of their way and not play normally to trigger a false positive to get a ban - then they've asked for it...

Especially since you need to kill a player 6-10 times in a row at minimum to get a ban... So it's not something you just do once and then get banned. You have to be persistent to get banned for this. So it's not really a concern.

Besides, you can turn it off if it bothers you, even tho I wouldn't recommend it (`lilac_aimbot_autoshoot 0`). :)

**Comment by crashzk** [comment]
I see, thanks for the explanation.
